evalml/tests/pipeline_tests/test_time_series_pipeline.py

```diff
@@ -1869,63 +1870,124 @@ def test_dates_needed_for_prediction_range(
 @pytest.mark.parametrize("set_coverage", [True, False])
 @pytest.mark.parametrize("add_decomposer", [True, False])
 @pytest.mark.parametrize("no_preds_pi_estimator", [True, False])
+@pytest.mark.parametrize("features", [True, False])
 def test_time_series_pipeline_get_prediction_intervals(
+    features,
     no_preds_pi_estimator,
     add_decomposer,
     set_coverage,
     ts_data_long,
+    ts_data_quadratic_trend,
 ):
-    X, _, y = ts_data_long
-    component_graph = {
-        "Time Series Featurizer": ["Time Series Featurizer", "X", "y"],
-        "DateTime Featurizer": ["DateTime Featurizer", "Time Series Featurizer.x", "y"],
-        "Drop NaN Rows Transformer": [
-            "Drop NaN Rows Transformer",
-            "DateTime Featurizer.x",
-            "y",
-        ],
-        "Regressor": [
-            "Exponential Smoothing Regressor"
-            if no_preds_pi_estimator
-            else "Linear Regressor",
-            "Drop NaN Rows Transformer.x",
-            "Drop NaN Rows Transformer.y",
-        ],
+    X, y = ts_data_quadratic_trend
+    if features:
+        component_graph = {
+            "DFS Transformer": ["DFS Transformer", "X", "y"],
+            "Drop NaN Rows Transformer": [
+                "Drop NaN Rows Transformer",
+                "DFS Transformer.x",
+                "y",
+            ],
+            "Regressor": [
+                "Exponential Smoothing Regressor"
+                if no_preds_pi_estimator
+                else "Linear Regressor",
+                "Drop NaN Rows Transformer.x",
+                "Drop NaN Rows Transformer.y",
+            ],
+        }
+        if add_decomposer:
+            component_graph.update(
+                {
+                    "STL Decomposer": [
+                        "STL Decomposer",
+                        "DFS Transformer.x",
+                        "y",
+                    ],
+                    "Drop NaN Rows Transformer": [
```

Comment:
do we add `Drop NaN Rows Transformer` twice usually?

Reply:
This isn't adding the `Drop NaN Rows Transformer` again - rather, it's updating the input X and y values. We only end up with one component after this call.